_source/logzio_collections/_log-sources/jaeger.md

```diff
@@ -84,14 +84,16 @@ logzio/jaeger-logzio:latest
 | Parameter | Description |
 |---|---|
 | ACCOUNT_TOKEN <span class="required-param"></span> | Required when using as a collector to ship traces to Logz.io. <br> {% include log-shipping/replace-vars.html token=true %} |
-| API_TOKEN	<span class="required-param"></span> | Required to read back traces from Logz.io. <br> Replace `<<API-TOKEN>>` with an [API token](https://app.logz.io/#/dashboard/settings/api-tokens) from the account you want to use. |
+| API_TOKEN <span class="required-param"></span> | Required to read back traces from Logz.io. <br> Replace `<<API-TOKEN>>` with an [API token](https://app.logz.io/#/dashboard/settings/api-tokens) from the account you want to use. |
 | REGION <span class="default-param">_Blank (US East)_</span> | Two-letter region code, or blank for US East (Northern Virginia). This determines your listener URL (where you're shipping the logs to) and API URL. <br> You can find your region code in the [Regions and URLs](https://docs.logz.io/user-guide/accounts/account-region.html#regions-and-urls) table. |
-| GRPC_STORAGE_PLUGIN_LOG_LEVEL	<span class="default-param">`warn`</span> | The lowest log level to send. From lowest to highest, log levels are `trace`, `debug`, `info`, `warn`, `error`. <br> Controls logging for Jaeger Logz.io Collector only (not Jaeger components). |
+| GRPC_STORAGE_PLUGIN_LOG_LEVEL <span class="default-param">`warn`</span> | The lowest log level to send. From lowest to highest, log levels are `trace`, `debug`, `info`, `warn`, `error`. <br> Controls logging for Jaeger Logz.io Collector only (not Jaeger components). |
+| COLLECTOR_ZIPKIN_HTTP_PORT | If you're using a Zipkin implementation to create traces, et this environment variable to `9411`. This tells Jaeger that it's accepting Zipkin spans so it can transform them to Jaeger-formatted spans. |
```

Comment:
```suggestion
| COLLECTOR_ZIPKIN_HTTP_PORT | If you're using a Zipkin implementation to create traces, set this environment variable to the HTTP port for the Zipkin collector service e.g. 9411 |
```

Reply:
@imnotashrimp do you want to mention that default value is 0?

Reply:
Not for the variable, no. As a note in the second column, I think that makes sense if you think it's useful to the user in this context.

Aside from that - would there be reason the user would choose a port besides 9411?

Reply:
@imnotashrimp if they already use that port. That's why I'm a bit afraid of telling them to just change it to 9411. In addition, they can set a different port in the collector and then they'll need to change the port in this flag as well

Reply:
@idohalevi Coming back to this — what's the right guidance here? I'm re-reading it and just as unsure as I was 28 days ago :/

Reply:
Let's do it today?

Reply:
אם לא עכשיו... :)